### DISCUSSION TITLE
How to solve the difference of PK curve obtained by setting different dosage simulation?

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1455] [Questions & Problems]
I am a beginner of PK-sim.The difference between the PK curve obtained when I administered the simulated drug at different doses (20-500mg) and the measured value is relatively large.The simulated value is much larger than the observed value when the dose of 20mg is set, and the simulated value is much smaller than the observed value when the dose of 500mg is set. What parameters are more critical when fitting PK curves of different doses?

**Comment by tobiasK2001** [comment]
Hi,
ab bit more details would help to answer your question. E.g. are you talking about IV or PO administration?

Best, Tobias

**Comment by tobiasK2001** [comment]
And, did you check out this tutorial? 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5080648/

Best, Tobias
